keps/sig-node/4191-split-image-filesystem/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Splitting the Image Filesystem
+kep-number: 4191
+authors:
+  - "@kannon92"
+owning-sig: sig-node
+participating-sigs:
+  - sig-aaa
```

Comment:
maybe `sig-storage` may be interested to review

Reply:
I'm not really sure they have much responsibility on this end.  If you know of someone who would be interested, please let me know.

The storage work would be in the container runtimes for docker and it wouldn't expect any of the PV/PVC work.

Reply:
added in case.